### DISCUSSION TITLE
.NET Core CLI 使用 firebird db 出現 Unable to load DLL 'fbembed' or one of its dependencies 信息

### DISCUSSION THREAD

**Discussion by netenken** [dotnetcore/FreeSql#827] [Q&A]
.NET Core CLI 使用 firebird db 出現 Unable to load DLL 'fbembed' or one of its dependencies 信息

不知道該將 fbembed.dll 相關庫, 放置於哪個目錄下 才能讓 CLI 順利運行

**Comment by netenken** [comment]
回來自問自答了, 原本 ConnectionString 我是使用 ServerType=1 也就是 embed dll 的...
昨晚想了下, 改成使用 ServerType=0  正規伺服器, 就可直接避開此一問題, 分享給大家..

**Comment by 2881099** [comment]
[freesql_firebird_embed.zip](https://github.com/user-attachments/files/18879676/freesql_firebird_embed.zip)
